pkgs/tools/text/chroma/default.nix

```diff
@@ -1,16 +1,32 @@
-{ lib, buildGoModule, fetchFromGitHub }:
+{ lib, buildGoModule, fetchgit, git }:
 
 buildGoModule rec {
   pname = "chroma";
-  version = "0.8.1";
+  version = "0.8.2";
 
-  src = fetchFromGitHub {
-    owner  = "alecthomas";
-    repo   = "chroma";
+  src = fetchgit {
+    url    = "https://github.com/alecthomas/chroma.git";
     rev    = "v${version}";
-    sha256 = "1gwwfn26aipzzvyy466gi6r54ypfy3ylnbi8c4xwch9pkgw16w98";
+    sha256 = "05dwp8l6wfyzxc84q4a4xydabx7x9k63x3mv7d8sjvcx6wzkp1i6";
+    leaveDotGit = true;
   };
 
+  nativeBuildInputs = [ git ];
```

Comment:
buildGoModule should already contain git as a nativeBuildInput.

Reply:
It doesn't.